crates/nu-command/src/example_test.rs

```diff
@@ -57,7 +57,10 @@ pub fn test_examples(cmd: impl Command + 'static) {
     };
 
     let cwd = std::env::current_dir().expect("Could not get current working directory.");
-    let _ = engine_state.merge_delta(delta, None, &cwd);
+
+    engine_state
+        .merge_delta(delta)
+        .expect("Error merging delta");
```

Comment:
all these expects will crash the shell if they happen.

Shouldn't we be a bit more resilient? Maybe just print an error and keep going?

Reply:
@jntrnr These are in a test code. I think it's OK to panic early instead of going with a wrong state when running tests.